ssz/sedes/serializable.py

```diff
@@ -0,0 +1,419 @@
+import abc
+import collections
+import copy
+import re
+
+from eth_utils import (
+    to_dict,
+    to_set,
+    to_tuple,
+)
+
+from ssz.constants import (
+    CONTAINER_PREFIX_LENGTH,
+)
+from ssz.exceptions import (
+    DeserializationError,
+    SerializationError,
+)
+
+
+class MetaBase:
+    fields = None
+    field_names = None
+    field_attrs = None
+
+
+def _get_duplicates(values):
+    counts = collections.Counter(values)
+    return tuple(
+        item
+        for item, num in counts.items()
+        if num > 1
+    )
+
+
+def validate_args_and_kwargs(args, kwargs, arg_names, allow_missing=False):
```

Comment:
I don't really like the name of the function as it's not validating the arguments itself, it's just checking that the function is called properly. But I don't have a better suggestion, so I'm fine with leaving it like this.